java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -0,0 +1,164 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.ZaddOptions;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Sorted set Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=sorted-set">Sorted set Commands</a>
+ */
+public interface SortedSetBaseCommands {
+    /**
+     * Adds members with their scores to the sorted set stored at <code>key</code>. If a member is
+     * already a part of the sorted set, its score is updated.
+     *
+     * @see <a href="https://redis.io/commands/zadd/">redis.io</a> for more details.
+     * @param key - The key of the sorted set.
+     * @param membersScoresMap - A mapping of members to their corresponding scores.
+     * @param options - The Zadd options.
+     * @param changed - Modify the return value from the number of new elements added, to the total
```

Comment:
```suggestion
     * @param key The <code>key</code> of the sorted set.
     * @param membersScoresMap A mapping of members to their corresponding scores.
     * @param options The Zadd options.
     * @param changed Modify the return value from the number of new elements added, to the total
```
ephasize `key`
remove dashes
repeat
Thanks